src/containers/createRoom.js

```diff
@@ -98,45 +117,3 @@ export class createRoom extends Component {
 }
 
 export default createRoom;
-
-{
```

Comment:
You'll want to start relying on version control to save old code for you. The trick is, you'll rarely end up going back for it, but if you do ever need it, it will be there.